cllc-public-app/Controllers/LoginController.cs

```diff
@@ -30,8 +30,8 @@ public LoginController(AppDbContext db, IConfiguration configuration, IHostingEn
         }
         
         [HttpGet]
+        [Route("headers")]
```

Comment:
Can you remove this Route line, as it changes the behavior of the controller.  We need the controller to respond to a GET request on /login in order for SiteMinder logins to work.